release/serve_tests/workloads/deployment_graph_long_chain.py

```diff
@@ -0,0 +1,207 @@
+        stats.append(end - start)
+
+    mean = round(np.mean(stats), 2)
+    std = round(np.std(stats), 2)
+    print("\t{} {} +- {} sec".format(name, mean, std))
+    return mean, std
+
+
+async def benchmark_batch_throughput(dag_handle, expected, num_calls=10):
+    """Enqueue all requests to dag handle and gather them all."""
+
+    async def run():
+        futures = []
+        for _ in range(num_calls):
+            futures.append(dag_handle.predict.remote(0))
+        results = await asyncio.gather(*futures)
+        for rst in results:
+            assert rst == expected
+
+    return await measure_throughput(
+        "benchmark_batch_throughput", run, multiplier=num_calls
+    )
+
+
+async def benchmark_latency(dag_handle, expected, num_calls=10):
+    """Call deployment handle in a blocking for loop."""
+
+    async def run():
+        for _ in range(num_calls):
+            assert await dag_handle.predict.remote(0) == expected
+
+    return await measure_latency("benchmark_batch_latency", run)
+
+
+@click.command()
+@click.option("--chain-length", type=int, default=4)
+@click.option("--init-delay-secs", type=int, default=0)
+@click.option("--compute-delay-secs", type=int, default=0)
+@click.option("--num-calls", type=int, default=10)
+@click.option("--local-test", type=bool, default=True)
+def main(
+    chain_length: Optional[int],
+    init_delay_secs: Optional[int],
+    compute_delay_secs: Optional[int],
+    num_calls: Optional[int],
+    local_test: Optional[bool],
+):
+    if local_test:
+        setup_local_single_node_cluster(4, num_cpu_per_node=2)
+    else:
+        setup_anyscale_cluster()
+
+    serve_dag = test_long_chain_deployment_graph(
+        chain_length,
+        init_delay_secs=init_delay_secs,
+        compute_delay_secs=compute_delay_secs,
+    )
+    dag_handle = serve.run(serve_dag)
+
+    # 1 + 2 + 3 + 4 + ... + chain_length
+    expected = ((1 + chain_length) * chain_length) / 2
+    assert ray.get(dag_handle.predict.remote(0)) == expected
+    loop = asyncio.get_event_loop()
+
+    throughput_mean, throughput_std = loop.run_until_complete(
+        benchmark_batch_throughput(dag_handle, expected, num_calls=num_calls)
+    )
+    latency_mean, latency_std = loop.run_until_complete(
+        benchmark_latency(dag_handle, expected, num_calls=num_calls)
+    )
+
+    results = {
+        "chain_length": chain_length,
+        "init_delay_secs": init_delay_secs,
+        "compute_delay_secs": compute_delay_secs,
+        "local_test": local_test,
+    }
+    results["perf_metrics"] = [
+        {
+            "perf_metric_name": "requests_per_s_avg",
+            "perf_metric_value": throughput_mean,
+            "perf_metric_type": "THROUGHPUT",
+        },
+        {
+            "perf_metric_name": "requests_per_s_std",
+            "perf_metric_value": throughput_std,
+            "perf_metric_type": "THROUGHPUT",
+        },
+        {
+            "perf_metric_name": "latency_per_s_avg",
```

Comment:
```suggestion
            "perf_metric_name": "latency_ms_avg",
```

not sure if it's actually in ms, but should be!